packages/api-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metriport/api-sdk",
-  "version": "7.15.2",
```

Comment:
That should be a `minor` update, since we're adding a new function, right?

Reply:
Yes -- wasn't sure what we liked to do. Will update.

Reply:
Done.